cli/src/test/java/hudson/cli/HexDumpTest.java

```diff
@@ -1,41 +1,49 @@
 package hudson.cli;
 
-import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.parallel.Execution;
 import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
 
 @Execution(ExecutionMode.CONCURRENT)
 public class HexDumpTest {
 
-  @Test
-  public void testToHex1() {
-    assertEquals("'fooBar'",
-            HexDump.toHex(new byte[] {'f', 'o', 'o', 'B', 'a', 'r'}));
-    assertEquals("0xc3",
-            HexDump.toHex(new byte[] {(byte)'Ã'}));
-    assertEquals("0xac '100'",
-            HexDump.toHex(new byte[] {(byte)'€', '1', '0', '0'}));
-    assertEquals("'1' 0xf7 '2'",
-            HexDump.toHex(new byte[] {'1', (byte)'÷', '2'}));
-    assertEquals("'foo' 0x0a\n'Bar'",
-            HexDump.toHex(new byte[] {'f', 'o', 'o', '\n', 'B', 'a', 'r'}));
+  @ParameterizedTest
```

Comment:
Optionally, you might consider including the expected value in the test name for easier diagnosis in case of failure.
```suggestion
  @ParameterizedTest(name = "expected: {0}")
```